content/blog/2025-01-dev-update/index.md

```diff
@@ -0,0 +1,63 @@
++++
+type = "blog"
+title = 'January 2025 Monthly Development Update'
+date = 2025-01-15
+draft = true
++++
+
+You're reading the TLA⁺ Foundation monthly development update.
+Here we summarize the past month of development for the benefit of Foundation patrons and interested members of the community.
+While things did slow down over the holidays compared to [the blockbuster December update](/blog/2024-12-dev-update/), we do have some material to cover.
+If your contribution was missed or some important part of it that wasn't captured in the summary, worry not!
+These newsletters will be published monthly so it's easy to hop on the next train; open an issue [here](https://github.com/tlaplus/foundation/issues).
+
+If you're interested in getting involved in the TLA⁺ community:
+- Learn TLA⁺ starting [here](https://lamport.azurewebsites.net/tla/learning.html)!
+- Read the mailing list [here](https://groups.google.com/g/tlaplus)!
+- Join the monthly virtual community meetings [here](https://groups.google.com/g/tlaplus/c/CpAEnrf-DHQ/m/YrORpIfSBwAJ)!
+- Start hacking on the tools themselves [here](https://github.com/tlaplus/tlaplus)!
+
+Let's start with some interesting non-coding-related developments & announcements:
+- The [TLA⁺ Community Event 2025](https://conf.tlapl.us/2025-etaps/) talk submission deadline is coming up soon, on February 7th.
+  The conference will be co-located with [ETAPS 2025](https://etaps.org/2025/) in Hamilton, Ontario, Canada on May 4th, 2025.
+  Get your submissions in soon!
+- East of the Atlantic, [ABZ 2025](https://abz-conf.org/site/2025/) - the 11th International Conference on Rigorous State-Based Methods - will take place from June 10th to 13th of 2025 in Düsseldorf, Germany.
+  TLA⁺-related submissions are welcome and the submission deadline is also fast approaching, on February 3rd.
+- [Murat Demirbas](https://cse.buffalo.edu/~demirbas/) wrote [a nice blog post](https://muratbuffalo.blogspot.com/2024/12/exploring-naiadclock-tla-model-in-tla.html) about using [tla-web](https://will62794.github.io/tla-web/#!/home?specpath=.%2Fspecs%2FTwoPhase.tla) to learn & explore an unfamiliar TLA⁺ spec.
+
+Now on to coding-related community developments:
+- [William Schultz](https://will62794.github.io/) has been doing some work on tla-web (to be filled in)
+- [Federico Ponzi](https://fponzi.me/) and [Hillel Wayne](https://www.hillelwayne.com/) landed a number of bugfixes in the TLA⁺ VS Code extension (to be filled in).
```

Comment:
I made pluscal show labels for macros, and also cleaned up the codebase a little